build.sbt

```diff
@@ -14,50 +14,52 @@
  */
 
 ThisBuild / organization := "za.co.absa"
-ThisBuild / name         := "spark-partition-sizing"
 
 lazy val scala211 = "2.11.12"
 lazy val scala212 = "2.12.12"
+lazy val spark2   = "2.4.7"
+lazy val spark3_2 = "3.2.3"
+lazy val spark3_3 = "3.3.1"
 
-import Dependencies._
+import SparkVersionAxis._
+import com.github.sbt.jacoco.report.JacocoReportSettings
 
-ThisBuild / crossScalaVersions := Seq(scala211, scala212)
 ThisBuild / scalaVersion := scala211
+ThisBuild / crossScalaVersions := Seq(scala211, scala212)
 
-import sbtrelease.ReleasePlugin.autoImport.ReleaseTransformations._
-
-releaseProcess := Seq[ReleaseStep](
-  checkSnapshotDependencies,
-  inquireVersions,
-  runClean,
-  runTest,
-  setReleaseVersion,
-  commitReleaseVersion,
-  tagRelease,
-  releaseStepCommand("publishSigned"),
-  setNextVersion,
-  commitNextVersion,
-  pushChanges
-)
```

Comment:
Do we not need the `releaseProcess` anymore?

Reply:
Weird, I thought I had removed it. It's definitely not in my local version (forgotten `push` perhaps?)
Anyway, will get rid of it.

Reply:
Eh, sorry. I read what I imagined instead what I saw...
I removed it as part of adding our release GitHub action - as the other `sbt` repos having - which I think wouldn't be compatible.
The release action has now moved to a follow up ticket (#36) and branch.